tools/toolchain/optimized_clang.sh

```diff
@@ -156,3 +156,6 @@ fi
 cd "${CLANG_BUILD_DIR}"
 ninja -C build install-distribution-stripped
 dnf remove -y clang clang-libs
+# above package removal might have removed those symbolic links, which will cause ccache not to work later on. Manually restore them.
+ln -s /usr/bin/ccache /usr/lib64/ccache/clang || true
+ln -s /usr/bin/ccache /usr/lib64/ccache/clang++ || true
```

Comment:
Does is work for aarch64 (does it use /usr/lib64 or /usr/lib?)

Why || true? It will ignore errors.

Reply:
The || true is in case the symbolic link exists.

Reply:
I did not test Arm. I assume it's less relevant (fewer folks compile on ARM), and I hope this change is not significant for ARM.

Reply:
It also means if we weren't successful in creating the link, we'll ignore the error. It's terrible practice, to ignore all errors when you want to ignore one.

Reply:
You can add the "-f" option to ln - it first deletes the destination and then creates the link - so it's allowed to run it multiple times, but another error in creating the link will be reported normally.
This "-f" has a different risk - of overwriting a file that wasn't supposed to be a link - but that's unlikely to happen in /usr/bin/ccache.

Reply:
OK, added a check ('test -h') if they exist prior to creating them.

Reply:
Better to us `ln -sf` than add a test.

Reply:
OK